modules/organisation/add-organisation.html

```diff
@@ -5,10 +5,10 @@
 	<div class="form-group">
 	<label for="orgName">Navn på organisasjon</label>
 	<input class="form-control" id="orgName" type="text" ng-model="addOrganisationController.orgName" />
-	<label for="orgShortName" type="text">Kortnavn uten mellomrom</label>
-	<input class="form-control" id="orgShortName" type="text" ng-model="addOrganisationController.orgShortName" />
+	<label for="orgShortName" type="text">Kortnavn uten mellomrom og andre spesialteikn</label>
+    <input class="form-control" id="orgShortName" type="text" pattern="[a-z\x00e6\x00f8\x00e5A-Z\x00c6\x00d8\x00c50-9]{3,}" ng-model="addOrganisationController.orgShortName" />
 	<button type="submit">Legg til</button>
```

Comment:
Heisann, takk for kjapp tilbakemelding :)

Er enig i at det gir falsk trygghet å berre leggje til denne typen validering i frontend klienten, diverre har ikkje eg god nok kompetanse på node js på å leggja dette inn i backend.

Eg fikk opp eit kjapt utviklingsmiljø og observerte at det stoppet å det så ut til å fungere for meg.

Helsing frå Torstein